src/passes/ConstHoisting.cpp

```diff
@@ -95,6 +95,8 @@ struct ConstHoisting : public WalkerPass<PostWalker<ConstHoisting>> {
         // v128 not implemented yet
         return false;
       }
+      case except_ref: // except_ref cannot have literal
```

Comment:
we usually have `{ }` around case bodies, but sadly not consistently. looks like we do on tihs particular switch though, so may be nice to keep it at least consistent.